internal/cli/cloudmanager/cloud_manager.go

```diff
@@ -16,6 +16,7 @@ package cloudmanager
 
 import (
 	"github.com/mongodb/mongocli/internal/cli/alerts"
+	"github.com/mongodb/mongocli/internal/cli/atlas/performanceadvisor"
```

Comment:
[nit] we should not introduce a dependency on atlas for cloud manager or ops manager, please consider taking `performanceadvisor` package to the same level (like the `events` package 

Also considering that for cloud manager and ops manager the option is a host id and not a process name, should we create just duplicate this but with the correct flag?